app/workers/pr_mergeability_checker.rb

```diff
@@ -19,6 +19,10 @@ def tag
     "<pr-mergeability-checker />"
   end
 
+  def unmergeable_comment
+    "#{tag}This pull request is not mergeable.  Please rebase and repush."
```

Comment:
t(w)oo (many) spaces here

Reply:
The change of this string will affect the older PR - ignoring them. It will be working only for new ones which will have this "new" string without "two" spaces due to string comparison at this [line](https://github.com/ManageIQ/miq_bot/pull/415/files#diff-26b156ca237a538a86e1df155e4d03ffR46). Are you sure about this?

Reply:
@europ I'd rather fix the line here, and leave the older PR comments as they are. But try asking in https://gitter.im/ManageIQ/miq_bot

Reply:
I'm fine with orphaning old comments but I am also fine with one or two spaces after a period.  It's cute to have the legacy of typewriters still around via the two space after periods  😆

Reply:
In other words, I'm more in favor of getting this merged than worrying about semantics.  :shipit:

Reply:
Agreed...only worry about the "new" comments moving forward.